build.gradle.kts

```diff
@@ -30,6 +31,7 @@ dependencies {
     runtimeOnly("com.h2database:h2")
     runtimeOnly("org.postgresql:postgresql")
     implementation("com.beust:klaxon:5.5")
+    implementation("org.jetbrains.kotlinx:kotlinx-serialization-json:1.2.2")
```

Comment:
Czy to jest trzecia biblioteka do serializacji? XD

Reply:
Tak, ale dodana przez Tomka. Ogólnie jeśli chodzi o kotlina, to dla mnie ta jest najlepsza, ale możemy się jeszcze zgadać (a nawet powinniśmy) i wybrać jedną. Pasuje na to taska dorzucić do improvementow.